package.json

```diff
@@ -38,28 +38,25 @@
     "node": ">=0.8"
```

Comment:
Let's consider bumping this to >=0.10 now.

And then we'll want a version bump here for release (could either be separate or part of this change). If we are bumping engines.node, we could bump the major version. Really no one uses 0.8 so it doesn't matter from that perspective. The changes in deps are a bigger change.  A new major ver is also a good time to change the "repository" field to point to joyent/node-fash.

Reply:
agreed on all of this... i'll make these changes.